app/views/components/docs/multi_section_viewer.yml

```diff
@@ -0,0 +1,26 @@
+name: Multi section viewer
+description: Switches between multiple sections of HTML
+body: |
+  This component is a wrapper for multiple section blocks. It ensures only one is shown at a time.
+
+  The sections can be defined and toggled statically, as in the examples. The module also exposes an API to show dynamic content in its own section.
+accessibility_criteria: |
+  The multi selection view box must:
+
+  - Notify the user when the content changes
+examples:
+  default:
+    embed: |
+      <button class="govuk-button" data-toggle="multi-section-viewer" data-target="sections" data-target-section="1">Show Section 1</button>
+      <button class="govuk-button" data-toggle="multi-section-viewer" data-target="sections" data-target-section="2">Show Section 2</button>
+      <%= component %>
+    data:
+      id: sections
+      block: |
+        <div class="app-c-multi-section-viewer__section" data-multi-section="1">
```

Comment:
It seems a bit unusual that the caller of this component needs to specify elements that are vital to the component `div.app-c-multi-section-viewer__section`. I don't think we have precedence for this and instead is normally done by passing arrays in.

Reply:
Good point - this was another hangover from it originally being part of the inline-image module. I've converted the component to pass this in with options, as for the tab component.